acct-user/gvm/gvm-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Greenbone vulnerability management program user"
+ACCT_USER_ID=495
+ACCT_USER_HOME=/var/lib/gvm
+ACCT_USER_SHELL=/bin/bash
```

Comment:
Why bash? Doesn't it really work with the default nologin?

Reply:
For the first time use we need to login as 'gvm' and have to download vulnerability feeds manually . Also we have to create certificates. Without that daemons will not start. 

```
su gvm
greenbone-nvt-sync
greenbone-scapdata-sync 
greenbone-certdata-sync
gvm-manage-certs
```

Reply:
Well, you can always use `su -s ...` but I don't have a strong opinion. I generally like the idea that service accounts don't have usable shell, just in case. @orlitzky, any opinion here?

Reply:
service accounts + rsyncing feeds

Technically you may right but when i packaging i always try to look from user side. An avarage linux user may confuse `su -s ` if try to setup package his/her own knowledge.  

Any harm with usable shell? If it is, i need to create detailed wiki page. This package is also complicated enough.